### PR TITLE
Add stubs for importlib.__init__

### PR DESCRIPTION
**Pull request by brettcannon**
Part of #189

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -59,3 +59,4 @@ analyze.py
 # Editor backup files
 *~
 .*.sw?
+.vscode/
```
Comment (gvanrossum) [inline .gitignore]: Instead, you should run
```
git config --global --add core.excludesfile ~/.gitignore_global
```
and create a `~/.gitignore_global` file that does this for all repos you use!

File: stdlib/3/importlib/__init__.pyi [hunk 2]
```diff
@@ -0,0 +1,23 @@
+import sys
+import types
+from typing import Any, Mapping, Optional, Sequence
+
+
```
Comment (gvanrossum) [inline stdlib/3/importlib/__init__.pyi]: In stubs we usually only leave one blank line between definitions (zero between methods in a class).

**Comment by gvanrossum** [comment]
Merging this as-is, if you feel compelled to change things you can do that in the next step. :-)

**Comment by brettcannon** [comment]
So I have a problem where mypy doesn't like the Liskov violation that was purposefully introduced into `importlib.abc` (it's all for minimal backwards-compatibility with the deprecated ABC). We have an ABC that defines the method:
```
@abc.abstractmethod
def find_module(self, fullname: str, path: Sequence[_Path] = None) -> Optional[Loader]: ...
```

One subclass overrides this method as:
```
def find_module(self, fullname: str, path: Optional[Sequence[_Path]]) -> Optional[Loader]: ...
```
Notice how `path` no longer has a default value.

The other subclass overrides the method as:
```
def find_module(self, fullname: str) -> Optional[Loader]: ...
```
Notice how `path` is no longer even accepted.

How do you want me to handle this to make mypy happy?

**Comment by gvanrossum** [comment]
Hm... Tricky. Maybe you can just leave the method out of the ABC
altogether? Or I would just allow certain calls that should be ruled out
and live with it.

Otherwise, experimentally it seems that mypy is happy with the subclass
that leaves path out altogether. But I don't understand the asymmetry, and
I wouldn't count on it.

I'm guessing https://github.com/python/mypy/issues/1237 is also relevant
here.

**Comment by brettcannon** [comment]
I'll just leave it out since the entire ABC has been deprecated since Python 3.4.

**Pull request merged**